analysis/active_analyses.R

```diff
@@ -609,18 +604,133 @@ for (c in cohorts) {
                          episode_event_threshold = episode_event_threshold,
                          covariate_threshold = covariate_threshold,
                          age_spline = TRUE,
-                         analysis = "sub_prior_parkinson_risk_false")
-  }
-    
+                         analysis = "sub_history_parkinson_risk_false")
   }
+}
 
 # Assign unique name -----------------------------------------------------------
 
 df$name <- paste0("cohort_",df$cohort, "-", 
                   df$analysis, "-", 
                   gsub("out_date_","",df$outcome))
 
-# Fix anxiety prior history variables ------------------------------------------
+# Remove covariate according to each outcome -----------------------------------
```

Comment:
I think we could simplify this. I am using cognitive impairment symptoms as an example here.

Instead of looping over each row of `df` as shown below:
```
for (i in 1:nrow(df)) {

  analysis <- df$analysis[i]
  outcome <- df$outcome[i]

  if (outcome == "out_date_cognitive_impairment_symptoms") {
    df$covariate_other[i] <- gsub("cov_bin_history_cog_imp_sympt;", "", df$covariate_other[i])
  }

}
```

We could edit all values of `df$covariate_other` based on the value in `df$outcome` at the same time:

```
df$covariate_other <- ifelse(df$outcome=="out_date_cognitive_impairment_symptoms",gsub("cov_bin_history_cog_imp_sympt;", "", df$covariate_other),df$covariate_other)
```

Reply:
Done.